overlord/snapstate/handlers.go

```diff
@@ -2379,7 +2379,7 @@ func installModeDisabledServices(st *state.State, snapst *SnapState, currentInfo
 	prevCurrentSvcs := map[string]bool{}
 	if psi := snapst.previousSideInfo(); psi != nil {
 		var prevCurrentInfo *snap.Info
-		if prevCurrentInfo, err = Info(st, snapst.InstanceName(), psi.Revision); prevCurrentInfo != nil {
+		if prevCurrentInfo, _ = Info(st, snapst.InstanceName(), psi.Revision); prevCurrentInfo != nil {
```

Comment:
it seems this should error for real, as if previousSideInfo is not nil, the Info should work, no?

Reply:
Indeed, I switched it to properly handling the error case now